src/ui/templates/cards/accordion.hbs

```diff
@@ -1,26 +1,38 @@
 <div class="yxt-AccordionCard js-yxt-AccordionCard{{#if isExpanded}} yxt-AccordionCard--expanded{{/if}}"
-  id="{{id}}">
-  <button class="yxt-AccordionCard-toggle js-yxt-AccordionCard-toggle"
-    aria-expanded="{{#if isExpanded}}true{{else}}false{{/if}}"
-    aria-controls="{{id}}"
-    tabindex="0"
-  >
-    <h3 class="yxt-AccordionCard-title">
-      {{{_config.title}}}
-    </h3>
-    <div class="yxt-AccordionCard-icon js-yxt-AccordionCard-icon{{#if isExpanded}} yxt-AccordionCard-icon--expanded{{/if}}"
-      data-component="IconComponent"
-      data-opts='{"iconName": "chevron"}'
-      data-prop="icon">
+id="{{id}}">
+
+  {{#if _config.details}}
+    <button class="yxt-AccordionCard-toggle js-yxt-AccordionCard-toggle"
+      aria-expanded="{{#if isExpanded}}true{{else}}false{{/if}}"
+      aria-controls="{{id}}"
+      tabindex="0"
+    >
+      <h3 class="yxt-AccordionCard-title">
+        {{{_config.title}}}
+      </h3>
+      <div class="yxt-AccordionCard-icon js-yxt-AccordionCard-icon{{#if isExpanded}} yxt-AccordionCard-icon--expanded{{/if}}"
+        data-component="IconComponent"
+        data-opts='{"iconName": "chevron"}'
+        data-prop="icon">
+      </div>
+    </button>
+  {{else}}
+    <div class="yxt-AccordionCard-toggle">
```

Comment:
minor nit: This class has `cursor: pointer` in its styling. Can this element have a normal cursor styling since clicking it does not do anything?